pages/using_elasticsearch.md

```diff
@@ -45,3 +52,30 @@ In production, JHipster expects an external Elasticsearch instance. By default,
 ### Using on Heroku
 
 On Heroku, the [Bonsai Elasticsearch](https://elements.heroku.com/addons/bonsai) is configured as an add-on. JHipster is automatically configured to talk to it. 
```

Comment:
Should we remove this? Or rephrase it to something like "In Heroku, JHipster used to configure Bonsai. Unfortunately..."

Reply:
Currently, it's accurate. I think we could change it, but we should change the code first.